tests/test_helpers.py

```diff
@@ -28,10 +28,14 @@ def test_get_users_from_file_no_file(self):
         with pytest.raises(FileNotFoundError):
             users = jccli_helpers.get_users_from_file(TEST_DATA_PATH+"invalid.json")
 
-    def test_get_users_from_file_users_in_data(self):
+    def test_get_users_from_json_file_users_in_data(self):
```

Comment:
you can remove the json specific tests since we are trusting that pyaml has already been tested for that functionality.  we care more about the content of the files than the format.